src/python/grpcio_tests/tests/unit/_metadata_test.py

```diff
@@ -19,6 +19,7 @@
 
 import grpc
 from grpc import _channel
+from grpc.experimental.aio._metadata import Metadata
```

Comment:
Let's expose `Metadata` as one of the public API.

Reply:
Done https://github.com/grpc/grpc/pull/22306/files#diff-8c5d8f70f661286cad86229d8f3ab036R72